### DISCUSSION TITLE
Where is the user dictionary?

### DISCUSSION THREAD

**Discussion by nikchha** [valentjn/vscode-ltex#130] [Q&A] [answered]
When I add a misspelled word to the dictionary via the _context menu_, it will no longer be marked as a spelling mistake by ltex.
However, it is not clear where I can revoke the addition to the user dictionary.
I checked the `ltex.workspaceDictionary` and `ltex.workspaceFolderDictionary` but those settings are empty in the `settings.json`. I did not find any further info in your (very good) documentation.

So my question is: Where can I edit dictionary words that were added via the _context menu_?

**Comment by carlosal1015** [comment, marked as answer]
Hi @nikchha, please look inside of `.vscode/` folder of your project.

You wonder where is this dictionary, but I founded (other) [dictionary](https://en.wikibooks.org/wiki/A-level_Computing/AQA/Paper_1/Fundamentals_of_data_structures/Dictionaries) with the key `ltex.workspaceFolderDictionary`

```json
{
    "ltex.workspaceFolderDictionary": {
        "en": [
            "foo"
        ]
    }
}
```

**Comment by valentjn** [comment]
Also, see the new section on [multi-scope settings](https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#multi-scope-settings) in the docs. In LT<sub>E</sub>X 8.0.0, the quick fix saves to an [external setting file](https://valentjn.github.io/vscode-ltex/docs/advanced-usage.html#external-setting-files) by default (see that link for info about the paths).
